Azure/azure-cli
# [Keyvault] `az keyvault create`, `az keyvault update`: add --public-network-access parameter

**Description**

Bump SDK version for KeyVault to 2021-06-01-preview. Add --public-network-access for `az keyvault create` and `az keyvault update`

**Testing Guide**

`az keyvault create -g {resource-group} -n {keyvault-name1} -l {location} --public-network-access 'Enabled'`
The publicNetworkAccess property should be 'Enabled'.
`az keyvault create -g {resource-group} -n {keyvault-name2} -l {location} --public-network-access 'Disabled'`
The publicNetworkAccess property should be 'Disabled'.
`az keyvault update -g {resource-group} -n {keyvault-name2} --public-network-access ‘Enabled'`
The publicNetworkAccess property should now be 'Enabled'.

**History Notes**


[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
